inference-engine/src/mkldnn_plugin/nodes/mkldnn_concat_node.h

```diff
@@ -29,7 +30,7 @@ class MKLDNNConcatNode : public MKLDNNNode {
     InferenceEngine::Precision getRuntimePrecision() const override;
 
 private:
-    size_t axis = 0;
+    size_t axis_ = 0;
```

Comment:
Please try to avoid such refactoring changes in the future. We don't have any rule regarding class members naming, so it is better to keep as is.

Reply:
As far as I know, we adhere to Google style. It says "Data members of classes, both static and non-static, are named like ordinary nonmember variables, but with a trailing underscore."

Reply:
As was discussed offline, there is no any rules in documentation, so code style changes were reverted.